src/JsonPath.Tests/ParsingTests.cs

```diff
@@ -139,4 +139,18 @@ public void ParseInOpWithoutOptions(string path)
 	{
 		Assert.Throws<PathParseException>(() => JsonPath.Parse(path));
 	}
+
+	[TestCaseSource(nameof(SuccessCases))]
+	public void ParseRelativeStarts(string path)
+	{
+		path = $"@{path[1..]}"; // Turn the absolute path into a relative path
+		Assert.DoesNotThrow(() => JsonPath.Parse(path, new PathParsingOptions(){AllowRelativePathStart = true}));
+	}
+
+	[TestCaseSource(nameof(SuccessCases))]
+	public void TryParseRelativeStarts(string path)
```

Comment:
I suppose I need to throw `.TryParse()` tests in here now :)

I have the test suite set up to run `.TryParse()` and then `.Parse()`, but it'd be good to do it here, too.